tuxemon/core/components/player.py

```diff
@@ -715,11 +714,16 @@ def get_adjacent_tiles(self, curr_loc, game):
 
 
 class Npc(Player):
-    def __init__(self, sprite_name="maple", name="Maple"):
-        # Initialize the parent menu class's default shit
-        Player.__init__(self, sprite_name, name)
+    def __init__(self, sprite_name="maple", slug="npc_maple"):
+        npcs = db.JSONDatabase()
+        npcs.load("npc")
+        npc_data = npcs['npc'][slug]
 
-        self.name = name
+        npc_name = trans(npc_data["name_trans"])

```

Comment:
Here, `trans` is not defined. To do translations, you first have to import the translator from the locale module:

```python
from core.components.locale import translator
```

Then we usually create an alias to the translator so lines aren't quite as long:

```python
        trans = translator.translate
```